osmo_jupyter/timezone.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+import pytz
+import pandas as pd
+
+
+OSMO_HQ_TIMEZONE = pytz.timezone('US/Pacific')
+SQL_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'  # Time format favored by mysql
+
+
+def to_aware(local_time):
+    """ get a timezone-aware object from local time string(s).
+
+    Args:
+        local_time: string of local time in any valid non-timezone-aware ISO format
+            Time should be in Osmo HQ local time.
+            eg. ('2012-01-01 12:00:00', '2012-01-01 12:00', '2012-01-01')
+    Returns:
+        timezone-aware datetime object
+    """
+    time = datetime.datetime.fromisoformat(local_time)
```

Comment:
hmmmm. this seems to be a python 3.7-only thing. currently it is possible to install python 3.7 in anaconda, but there isn't a distribution that comes with it preinstalled. 
input needed: should I:
1. do this using dateutil instead, which will allow a wider range of dates (too permissive, really - it'll make a "guess" on some crazy stuff) or
2. take a dependency on [iso8601](https://pypi.org/project/iso8601/) and use that, or
3. just "hold the line" on python 3.7 and teach people how to upgrade

Reply:
3) seems like it could drop down a rabbit hole in helping with individual configurations for 8 people and the other issues that might arise with using python 3.7 within anaconda.

For 1) Can you give an example of what it would guess on?

For now I'd vote for option 2 given what seems like reliable functionality (cursory glance) and doesn't probably introduces less "unknown unknowns".